<ui version="4.0">
 <class>MenuEstudiante</class>
 <widget class="QMainWindow" name="MenuEstudiante">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>900</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>MenULE - Menú Estudiante</string>
  </property>
  <property name="windowState">
   <set>Qt::WindowFullScreen</set>
  </property>
  <property name="styleSheet">
   <string>
QMainWindow {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                                            stop:0 #1565c0, stop:1 #90caf9);
}
QLabel#labelUsuario {
        font-size: 20px;
        color: white;
        font-weight: bold;
        padding: 8px;
}
QCalendarWidget {
        background-color: white;
        border-radius: 8px;
}
QPushButton {
        background-color: #1976d2;
        color: white;
        font-weight: bold;
        border-radius: 8px;
        padding: 6px;
}
QPushButton:disabled {
        background-color: #90a4ae;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <widget class="QLabel" name="labelUsuario">
      <property name="text">
       <string>Bienvenido, Usuario</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contentLayout">
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
      <item>
       <widget class="QWidget" name="contenedorMenuDerecha">
        <layout class="QVBoxLayout" name="verticalLayoutDerecha">
         <item>
          <widget class="QLabel" name="labelPrimeros">
           <property name="text">
            <string>Primeros platos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listaPrimeros"/>
         </item>
         <item>
          <widget class="QLabel" name="labelSegundos">
           <property name="text">
            <string>Segundos platos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listaSegundos"/>
         </item>
         <item>
          <widget class="QLabel" name="labelPostres">
           <property name="text">
            <string>Postres</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listaPostres"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnVisualizarMenu">
        <property name="text">
         <string>Visualizar Menú</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReservarComida">
        <property name="text">
         <string>Reservar</string>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVolver">
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>